app/src/main/java/com/javahelp/frontend/fragments/AccountFragment.java

```diff
@@ -0,0 +1,66 @@
+package com.javahelp.frontend.fragments;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.javahelp.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link AccountFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class AccountFragment extends Fragment {
+
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
+    private static final String ARG_PARAM2 = "param2";
+
+    // TODO: Rename and change types of parameters
+    private String mParam1;
+    private String mParam2;
```

Comment:
Comments should be added to make it clear that this is example code & boilerplate, and that this will be removed and filled in with content later.

Reply:
ok